pandas/_libs/groupby_helper.pxi.in

```diff
@@ -29,10 +29,10 @@ def get_dispatch(dtypes):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
```

Comment:
A challenge in this file if you’re up to it: use fused types instead of tempita.  Last time I tried I got weird errors that I never figured out.

Reply:
I think it can be done. Can I do it in another PR after this is (hopefully) merged?

Reply:
Yah, definitely separate PR.